wd_demo.au3

```diff
@@ -798,4 +798,19 @@ Func SetupEdge($bHeadless)
 EndFunc   ;==>SetupEdge
 
 Func SetupOpera($bHeadless)
-EndFunc   ;==>SetupEdge
+	#forceref $bHeadless
+	_WD_Option('Driver', 'operadriver.exe')
+	_WD_Option('Port', 9515)
+	_WD_Option('DriverParams', '--verbose --log-path="' & @ScriptDir & '\opera.log"')
+
+;~ 	Local $sCapabilities = '{"capabilities": {"alwaysMatch": {"goog:chromeOptions": {"w3c": true, "excludeSwitches": [ "enable-automation"]}}}}'
+	_WD_CapabilitiesStartup()
+	_WD_CapabilitiesAdd('alwaysMatch', 'opera')
+	_WD_CapabilitiesAdd('w3c', True)
+	_WD_CapabilitiesAdd('excludeSwitches', 'enable-automation')
+	_WD_CapabilitiesAdd('binary', 'C:\Users\Szef\AppData\Local\Programs\Opera\opera.exe')
```

Comment:
currently I'm wondirng how to find the path .
There is a need to find speciall function for this.

Reply:
maybe we can create function `_WD_GetBrowserPath($sBrowser)` similar to `_WD_GetBrowserVersion($sBrowser)`
I do review and I found:
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe`
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\firefox.exe`
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\msedge.exe`
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\IEXPLORE.EXE`

but for `Opera `
`HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\opera.exe`

What you @Danp2  and @Sven-Seyfert  think about adding `_WD_GetBrowserPath($sBrowser)` and about the `Opera` registry entries.

Reply:
> about the Opera registry entries.

This doesn't match my experience. I recently installed Opera via Chocolatey, and the registry key is under `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\`. I suspect the difference is related to how the software was installed (ie: everyone vs individual)

> think about adding _WD_GetBrowserPath($sBrowser) 

I'm good with this. `_WD_GetBrowserVersion` already looks up this information, so the code can be moved from there into a separate function. The new function can check both registry locations.

Reply:
Have you tried the 64 bit version of Operadriver? I don't think you need the `binary` entry with it.

Reply:
I have 64bit vesion
![image](https://user-images.githubusercontent.com/11089482/155717759-e82aba6a-93ed-4293-8b50-f1343a8808db.png)

Reply:
64 bit version of Operadriver, not Opera

Reply:
ah.... it was 32bit
Please check also: [_WD_Startup: $sDriverBitness information](https://github.com/Danp2/au3WebDriver/pull/224)

Reply:
After checking with 64 bit driver.

I can say that using 32bit driver requires to set capabilities:
```autoit
```
and using 64bit driver dosen't require to set this capability, but at the same time setting is not affecting the script.
So this is good habbit to setup for any case.